lint-staged.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
-  "*.{js,ts,tsx,.graphql,.gql}": ["yarn eslint-staged", "yarn prettier"],
-  "*.{ts,tsx}": () => "tsc -p tsconfig.json --noEmit",
+  "*.{js,ts,tsx}": ["yarn eslint-staged", "yarn prettier"],
+  "*.{graphql,gql}": "yarn prettier --parser graphql",
```

Comment:
What does this do?

Reply:
This ensures graphql queries are properly formatted in the pre commit hook it wasn't working before.